### PR TITLE
several increase in the same block

### PR DESCRIPTION
**Pull request by jo-tm**
to fix https://github.com/hirosystems/stacks-2-1-testing/issues/30

### CODE REVIEW

File: tests/integration/pox/stacking/direct-stacking-many-increase.spec.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+    const aliceTotalUstx = alicePoxAddressInfo
+      ? cvToString(alicePoxAddressInfo["total-ustx"])
+      : "";
+    const bobTotalUstx = bobPoxAddressInfo
+      ? cvToString(bobPoxAddressInfo["total-ustx"])
+      : "";
+
+    // The total-ustx values should be different due to the bug in stack-increase
+    expect(aliceTotalUstx).not.toEqual(bobTotalUstx);
```
Comment (friedger) [inline tests/integration/pox/stacking/direct-stacking-many-increase.spec.ts:138]: I like positive assertions. Can you assert the exact value for both?

**Review by friedger — APPROVED** [review]
Looks good.

**Review by obycode — APPROVED** [review]
👍

**Comment by friedger** [comment]
format check failed.
I'll merge nevertheless to continue refactoring

**Pull request merged**